src/components/Pagination/Pager.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { noop } from '../../common/helpers';
+
+/**
+ * Pager component for Patternfly React
+ */
+const Pager = ({
+  baseClassName,
+  className,
+  messages,
+  disableNext,
+  onNextPage,
+  disablePrevious,
+  onPreviousPage
+}) => {
+  const classes = cx('pager', className);
+  return (
+    <ul className={classes}>
+      <li className={disablePrevious ? 'previous disabled' : 'previous'}>
+        <a
```

Comment:
I know Patternfly uses anchors but I think we should use buttons. Using `<Button bsStyle="link" >` provides accessibility and putting `disabled` on the button uses the disabled cursor and doesn't show the clicks.

We may need to add scss/less to do this but I think it's the right thing to do. Thoughts? I think we should open an issue against Patternfly to support the use of buttons in this context.

Reply:
Whether its buttons or anchors, the issue (of disabled buttons not showing appropriate cursor type) will persist so long as this thinggah lurks about https://github.com/patternfly/patternfly/blob/master/src/less/pager.less#L35

Reply:
SO we can open an issue to fixup the cursor type on disabled.. buttttt doesn't necessarily require the change from anchors to buttons 🤔

Reply:
Putting disabled on a button shows the disabled cursor. 

Putting disabled on the anchor rather than just on the li actually disables the anchor (no click action). Adding an href would also address the accessibility. At a minimum, we should do that.

Reply:
I've updated the storybook so yah can see, putting the disabled on the anchor does not remedy the situation as the aforementioned  class overrides with `cursor: default;`

Reply:
correct, no cursor change but it at least it doesn't handle the click (same should be done in pagination row for #253)

Reply:
? I'm confused, it didn't handle the click before because of line 27, the presence or absence of `disabled` has no bearings on the triggering of an event 😕

Reply:
OK, you're right. I was being mislead by my eyes and the fact that selecting the `Previous button disabled` knob is not disabling the previous button.

Reply:
OH THE KNOB ISN'T WORKING great catch, on it.